### ISSUE TITLE
git 2.36.1

### ISSUE THREAD

**Issue by dwirtz** [chocolatey-community/chocolatey-packages#1911]
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

2.36.1

### Download location

https://github.com/git-for-windows/git/releases/download/v2.36.1.windows.1/PortableGit-2.36.1-64-bit.7z.exe

### Package Page

https://community.chocolatey.org/packages/git

**Comment by pascalberger** [comment]
Chocolatey reports a conflict when pushing 2.36.1 to the repo: https://gist.github.com/choco-bot/a14b1e5bfaf70839b338eb1ab7f8226f#ignored

@pauby Do you have any idea why we retrieve a 409 here? There doesn't seem to be a version 2.36.1 already pushed.

**Comment by AdmiringWorm** [comment]
> Just checking, is the description issue you mention the reason that the upgrade fails ?
> 
> I just ran `choco upgrade git -y` and got the following:
> 
> ```
> ERROR: Running ["C:\ProgramData\chocolatey\lib\git.install\tools\Git-2.36.1-64-bit.exe" /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /NOCANCEL /SP- /LOG /COMPONENTS="icons,assoc,assoc_sh,,,,gitlfs,icons\quicklaunch" /o:PathOption=CmdTools ] was not successful. Exit code was '1'. See log for possible error messages.
> 
> The upgrade of git.install was NOT successful.
> Error while running 'C:\ProgramData\chocolatey\lib\git.install\tools\chocolateyInstall.ps1'.
>  See log for details.
> ```

No, the description issue is only prevent packages being pushed to chocolatey.org.
In your case it is the installer itself that the package uses that complains, unfortunately an exit code of 1 is too generic to be of any help here, but the log file may contain information about why the installer complains.

You should be able to find the log file located in `%TEMP%\chocolatey` with a name that starts with `Setup Log`.
If you open that hopefully it will contain information about why it didn't install.
